data/crac/crac-impl/src/main/java/com/farao_community/farao/data/crac_impl/json/deserializers/UsageRuleDeserializer.java

```diff
@@ -80,19 +80,14 @@ private static FreeToUseImpl deserializeFreeToUseUsageRule(JsonParser jsonParser
                     break;
 
                 case INSTANT:
-                    instantId = jsonParser.nextTextValue();
+                    jsonParser.nextToken();
+                    instant = jsonParser.readValueAs(Instant.class);
                     break;
 
                 default:
                     throw new FaraoException(UNEXPECTED_FIELD + jsonParser.getCurrentName());
             }
         }
-
-        Instant instant = simpleCrac.getInstant(instantId);
-        if (instant == null) {
```

Comment:
I think we should keep this check
can't it happen, for example, that the usageRule references OUTAGE instant, and that the json's instants field only contains PREVENTIVE and CURATIVE ?

Reply:
it cannot happen with the latest changes, all instants of the enum implictely exist in the Crac now